packages/shared/src/components/cards/PostMetadataReadingHistory.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement, ReactNode } from 'react';
```

Comment:
I think you copied this file from the extension right?
This is not really specific to reading history.

I would suggest completely moving this to the shared components and removing the one in the extension.
(Side benefit: I also need this for the companion app)

Reply:
I copied it from /shared/src/components/cards/PostMetadata.tsx, just created another one for Reading history item. It is already in the shared folder 🤔

Reply:
I agree with Chris. If this is displaying the same thing, let's use one component. I think the only change you need will be the type of `post` to accommodate the two scenarios.